server.go

```diff
@@ -155,13 +166,12 @@ func (s *Server) Listen() {
 	}
 
 	go func() {
-		s.statusLogger.Info("starting server on " + s.addr)
```

Comment:
Please check the line I am refererring to. Your answer doesn't make sense for. Why don't we want to print the listening host/port anymore?